overlord/state/task.go

```diff
@@ -465,14 +465,24 @@ func (t *Task) At(when time.Time) {
 	}
 }
 
+// TaskSetEdge allows marking tasks inside a TaskSet
+type TaskSetEdge string
+
 // A TaskSet holds a set of tasks.
 type TaskSet struct {
 	tasks []*Task
+
+	edges map[TaskSetEdge]*Task
 }
 
 // NewTaskSet returns a new TaskSet comprising the given tasks.
 func NewTaskSet(tasks ...*Task) *TaskSet {
-	return &TaskSet{tasks}
+	return &TaskSet{tasks, nil}
+}
+
+// Edge returns the task marked with the given TaskSetEdge
```

Comment:
missing period.

Reply:
There was one more missing period (outside of my PR) that I fixed opportunistly.